### ISSUE TITLE
feat: error when port 4001 is already in use

### ISSUE THREAD

**Issue by DirtyF** [tinacms/tinacms#1969]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Given I already run a tina instance in another terminal
When I run yarn dev
Then I get an error: 

```
yarn run v1.22.10
$ yarn tina-gql server:start -c "next dev"
$ /Users/frank/github/tina/blog-starter-app/node_modules/.bin/tina-gql server:start -c 'next dev'

events.js:353
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::4001
    at Server.setupListenHandle [as _listen2] (net.js:1318:16)
    at listenInCluster (net.js:1366:12)
    at Server.listen (net.js:1452:7)
    at start (/Users/frank/github/tina/blog-starter-app/node_modules/tina-graphql-gateway-cli/dist/index.js:660:18)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
Emitted 'error' event on Server instance at:
    at emitErrorNT (net.js:1345:8)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) {
  code: 'EADDRINUSE',
  errno: -48,
  syscall: 'listen',
  address: '::',
  port: 4001
}
```

### Expected Behavior

A simpler message like: Port 4001 already in use.

### Steps To Reproduce

1. Run yarn dev in a tina cloud project
2. Run yarn dev in another tina cloud project

### Environment

```markdown
- Browser: Firefox
- OS: macOS
- Node: 14
- npm: 7
```

### Anything else?

Not critical.

**Comment by CatalinCsnMaster** [comment]
I can take this issue 😁

**Comment by logan-anderson** [comment]
@CatalinCsnMaster That would be awesome! The code for the CLI can be found [here](https://github.com/tinacms/tinacms/tree/main/packages/%40tinacms/cli). Please let us know if you have any issues contributing. 

The code for each command can be found here: https://github.com/tinacms/tinacms/tree/main/packages/%40tinacms/cli/src/cmds

I have a feeling the error is coming from the server start command.

**Comment by CatalinCsnMaster** [comment]
@logan-anderson I got a little confused with the changeset. 
But I figured it out and next time the pr will be prettier 😃

**Comment by DirtyF** [comment]
Fixed in #2047
